db_stress_tool/db_stress_gflags.cc

```diff
@@ -186,6 +186,9 @@ DEFINE_int64(compressed_cache_size, -1,
 DEFINE_int32(compaction_style, ROCKSDB_NAMESPACE::Options().compaction_style,
              "");
 
+DEFINE_int32(num_levels, ROCKSDB_NAMESPACE::Options().num_levels,
```

Comment:
Maybe add this to db_stress_common.h as well.

Reply:
Done.